ui/src/app/utils/access-utils.tsx

```diff
@@ -308,14 +308,6 @@ export const getAccessModuleConfig = (
   );
 };
 
-// the modules subject to Registered Tier Annual Access Renewal (AAR), in the order shown on the AAR page.
-export const rtAccessRenewalModules = [
```

Comment:
I'd prefer not to hard-code these if possible; fortunately there was only one code use, `hasRtExpired` below, and it could be updated to use config values

Reply:
(I'm happy to hard-code these for tests!)